frontend/helpers/next-helpers.ts

```diff
@@ -18,8 +18,9 @@ interface WithLoggingMiddleware {
 export const withLogging: WithLoggingMiddleware =
    ({ pageName }) =>
    (next) => {
-      return async (context) => {
-         return timeAsync(`page.${pageName}.server_side_props`, async () => {
+      return withTiming(
```

Comment:
It looks like `timeAsync` wraps `withTiming`, is there a reason to change the use of `timeAsync`?

Reply:
I created `withTiming` to match a more composable pattern, requiring one less closure. `timeAsync()` is still for single parameter-less function invocations if that's more convenient, but here using `withTiming` results in a cleaner callsite. Note that `timeAsync` doesn't *wrap* `withTiming`, it simply calls it an invokes the return value immediately.